snuba/settings/__init__.py

```diff
@@ -56,6 +56,9 @@
     "subscriptions.process_message": 0.1,
 }
 
+CLICKHOUSE_READONLY_USER = os.environ.get("CLICKHOUSE_READONLY_USER", "readonly")
```

Comment:
I think we should leave this as `default` and set to `readonly` in those environments where we have explicitly added a user with this name otherwise it will fail

Reply:
Sure, does this mean we have to wait on ops to test the tool, though?

Reply:
We'll need to open a PR in ops to set `CLICKHOUSE_READONLY_USER="readonly"`. In the meantime it would use the default user.